regnety/config/config.py

```diff
@@ -52,71 +148,124 @@ def get_model_config(flops: str):
-            bottleneck_ratio = 1,
-            SE_ratio = 0.25,
-            wa = 36,
-            w0 = 24,
-            wm = 2.5
+            name="RegNetY 200MF",
+            flops="200MF",
+            num_classes=1000,
+            depths=[1, 1, 4, 7],
+            widths=[24, 56, 152, 368],
+            group_width=8,
+            bottleneck_ratio=1,
+            SE_ratio=0.25,
+            wa=36,
+            w0=24,
+            wm=2.5
         )
-        
+
     elif flops.lower() == '400mf':
         return RegNetYConfig(
-            name = "RegNetY 400MF",
-            flops = "400MF",
-            num_classes = 1000,
-            depths = [1, 3, 3, 6],
-            widths = [48, 104, 208, 440],
-            group_width = 8,
-            bottleneck_ratio = 1,
-            SE_ratio = 0.25,
-            wa = 28,
-            w0 = 48,
-            wm = 2.1
+            name="RegNetY 400MF",
+            flops="400MF",
+            num_classes=1000,
+            depths=[1, 3, 3, 6],
+            widths=[48, 104, 208, 440],
+            group_width=8,
+            bottleneck_ratio=1,
+            SE_ratio=0.25,
+            wa=28,
+            w0=48,
+            wm=2.1
         )
-    
+
     elif flops.lower() == '600mf':
         return RegNetYConfig(
-            name = "RegNetY 600MF",
-            flops = "600MF",
-            num_classes = 1000,
-            depths = [1, 3, 7, 4],
-            widths = [48, 112, 256, 608],
-            group_width = 16,
-            bottleneck_ratio = 1,
-            SE_ratio = 0.25,
-            wa = 33,
-            w0 = 48,
-            wm = 2.3
+            name="RegNetY 600MF",
+            flops="600MF",
+            num_classes=1000,
+            depths=[1, 3, 7, 4],
+            widths=[48, 112, 256, 608],
+            group_width=16,
+            bottleneck_ratio=1,
+            SE_ratio=0.25,
+            wa=33,
+            w0=48,
+            wm=2.3
         )
-    
+
     elif flops.lower() == '800mf':
         return RegNetYConfig(
-            name = "RegNetY 800MF",
-            flops = "800MF",
-            num_classes = 1000,
-            depths = [1, 3, 8, 2],
-            widths = [64, 128, 320, 768],
-            group_width = 16,
-            bottleneck_ratio = 1,
-            SE_ratio = 0.25,
-            wa = 39,
-            w0 = 56,
-            wm = 2.4
+            name="RegNetY 800MF",
+            flops="800MF",
+            num_classes=1000,
+            depths=[1, 3, 8, 2],
+            widths=[64, 128, 320, 768],
+            group_width=16,
+            bottleneck_ratio=1,
+            SE_ratio=0.25,
+            wa=39,
+            w0=56,
+            wm=2.4
         )
+
+
+def get_train_config():
+    """
+    Getter function for training config. Config is same as in the paper
+    (see link above). If ambiguous, 
+    https://github.com/facebookresearch/pycls/blob/master/pycls/core/config.py  
+    is assumed to be source of truth.
+
+    Args: None
```

Comment:
Please update

Reply:
I'm sorry, I didn't get what should be updated. Could you please elaborate?

Reply:
Sorry, please ignore.